packages/editor/src/browser/editor-navigation-contribution.ts

```diff
@@ -91,6 +99,25 @@ export class EditorNavigationContribution implements Disposable, FrontendApplica
             execute: () => this.toggleWordWrap(),
             isEnabled: () => true,
         });
+        this.commandRegistry.registerHandler(EditorCommands.REOPEN_CLOSED_EDITOR.id, {
+            execute: () => this.reopenLastClosedEditor()
+        });
+    }
+
+    async reopenLastClosedEditor(): Promise<void> {
```

Comment:
Perhaps the method should be document to state that it will attempt to re-open the last closed editor if possible, and if it cannot it will attempt to reopen the next item in the stack.